decidim-sortitions/app/views/decidim/sortitions/admin/sortitions/confirm_destroy.html.erb

```diff
@@ -6,15 +6,11 @@
   </h2>
 </div>
 
-<div class="item__edit item__edit-1col">
+<div>
```

Comment:
Let's leave the 1col design, as its a bit better in large screens and more consistent with the rest of the UI 

```suggestion
<div class="item__edit item__edit-1col">
```